scripts/classes/settings.gd

```diff
@@ -0,0 +1,35 @@
+class_name GameSettings
```

Comment:
Since this doesn't extend Node, it isnt an object in scene, so you can't use ``@export``s there.
Since this doesn't extend Resource, it isnt a resource in game files, so you can't use ``@export``s there.
This is being created in ``game.gd`` through
```gdscript
settings = GameSettings.new(DEBUG, gamemode, next_scene)
```
Then what is the point of exports on this class?
Exports are useful for creating new assets, but this stuff is basically hardcoded.
Creation of this in ``game.gd`` does take exports of itself into account, but then why not ``@export settings: GameSettings`` and have GameSettings be a Resource?

Reply:
Made these changes in #31 .